### PR TITLE
bump flow and Fixes #1585

### LINKED ISSUE #1585: Textarea - type props
replace `Type` with just a `type`

https://github.com/brainly/style-guide/blob/36f3da4b59595ab7edc7ea5cf8af7dffffe30fec/src/components/form-elements/Textarea.jsx#L20

### PR DESCRIPTION
**Pull request by PinkaminaDianePie**

### CODE REVIEW

File: src/components/form-elements/Textarea.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const SIZE = {
 };
 
 export type TextareaPropsType = {
-  Type?: string | () => React.Node,
+  Type?: string | any => React$Node,
```
Comment (burakukula) [inline src/components/form-elements/Textarea.jsx]: @PinkaminaDianePie Bartek added this issue lately - https://github.com/brainly/style-guide/issues/1585, could you have a look?

Comment (PinkaminaDianePie) [reply src/components/form-elements/Textarea.jsx]: done

Comment (burakukula) [reply src/components/form-elements/Textarea.jsx]: Awesome, thx. If you add `Fixes #1585` it should automatically close the issue as soon as you merge. 🙃

Comment (PinkaminaDianePie) [reply src/components/form-elements/Textarea.jsx]: add it where? commit message? pr name?

**Pull request merged**